src/ErrorReporting/V1beta1/ErrorGroupServiceGrpcClient.php

```diff
@@ -0,0 +1,64 @@
+<?php
+// GENERATED CODE -- DO NOT EDIT!
+
+// Original file comments:
+// Copyright 2016 Google Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+namespace Google\Cloud\ErrorReporting\V1beta1 {
```

Comment:
The Grpc Client classes are coming from different generation code - the grpc plugin rather than protobuf. I will look at what is required to make them consistent, including fixing the doc block issue.